Assets/_BForBoss/_Analytics/Scripts/PerigonAnalytics.cs

```diff
@@ -57,16 +64,31 @@ public void EndSession()
             Mixpanel.Track(SessionEnd);
             Mixpanel.Flush();
         }
-        
-        public void LogDeathEvent(String name)
+
+        #region CUSTOM EVENTS
+
+        public void LogDeathEvent(String deathAreaName)
         {
             var props = new Value();
             
             props[EventAttribute.Course] = EventConstant.RaceCourse;
-            props[EventAttribute.Name] = name;
+            props[EventAttribute.Name] = deathAreaName;
             Mixpanel.Track(Event.PlayerDeath, props);
         }
 
+        public void LogCheckpointEvent(float time, String checkpointName)
+        {
+            String formattedTime = string.Format($"{time:0.00}s");
```

Comment:
hmm sorry. now that i think about it, can you set the time to be a float? 

That way we can actually chart it.